src/main/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorUtils.java

```diff
@@ -777,487 +901,141 @@ public static String createSpliceSiteCodonChange(final int variantStart,
     }
 
     /**
-     * Creates the string representation of the codon change for the given {@link SequenceComparison}.
+     * Creates the string representation of the protein change for the given {@link SequenceComparison}.
+     *
+     * Specifically, this creates a rendering of the first amino acids to be changed by a variant.
+     *
      * Requires that the given {@code seqComp} has the following fields defined with values that are not {@code null}:
-     *     referenceAminoAcidSequence
-     *     proteinChangeStartPosition
-     *     proteinChangeEndPosition
-     *     alternateAminoAcidSequence
+     *     proteinChangeInfo
      *     referenceAllele
      *     alternateAllele
-     * In the case of an insertion, the given {@code seqComp} must have the following fields defined with values that are not {@code null}:
-     *     codingSequenceAlleleStart
-     *     transcriptCodingSequence
-     *     strand  (must also not be {@link Strand#NONE}
+     *     alleleStart
      * @param seqComp {@link SequenceComparison} representing the alternate and reference alleles for a DNA sequence.  Must not be {@code null}.
+     * @param startCodon {@link Locatable} representing the start codon for the coding region for the variant in {@code seqComp}.  May be {@code null}.
      * @return A {@link String} representing the codon change for the given {@link SequenceComparison}.
      */
-    public static String getProteinChangeString(final SequenceComparison seqComp) {
-
-        Utils.nonNull(seqComp);
-        Utils.nonNull(seqComp.getReferenceAminoAcidSequence());
-        Utils.nonNull(seqComp.getProteinChangeStartPosition());
-        Utils.nonNull(seqComp.getProteinChangeEndPosition());
-        Utils.nonNull(seqComp.getAlternateAminoAcidSequence());
+    public static String renderProteinChangeString(final SequenceComparison seqComp,
```

Comment:
I like the idea of refactoring to make `ProteinChangeInfo` smarter, but since a lot of the data required are in the `SequenceComparison` object, I'm going to leave it as-is for now.